src/test/kotlin/no/nav/personbruker/dittnav/eventtestproducer/ytelsestesting/TestDataServiceTest.kt

```diff
@@ -0,0 +1,40 @@
+package no.nav.personbruker.dittnav.eventtestproducer.ytelsestesting
+
+import no.nav.personbruker.dittnav.eventtestproducer.beskjed.BeskjedProducer
+import no.nav.personbruker.dittnav.eventtestproducer.config.Environment
+import no.nav.personbruker.dittnav.eventtestproducer.done.DoneProducer
+import no.nav.personbruker.dittnav.eventtestproducer.innboks.InnboksProducer
+import no.nav.personbruker.dittnav.eventtestproducer.oppgave.OppgaveProducer
+
+internal class TestDataServiceTest {
+
+    private val env = createPropertiesForTestEnvironment()
+
+    private val doneProducer = DoneProducer(env)
+    private val beskjedProducer = BeskjedProducer(env)
+    private val innboksProducer = InnboksProducer(env)
+    private val oppgaveProducer = OppgaveProducer(env)
+
+//    @Test
```

Comment:
Skal denne være med?

Reply:
Ja, den er med slik at ikke test-eventene blir forsøk produsert hver gang man bygger appen. Man kan jo ta hele linja bort, men har det med som utkommentert for å gi et hint om hvordan det kan brukes.